cloudhsm/base-images/jdk11-jce-gradle/Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Base builder image to build apps that need the HSM
+
+ARG JAVA_HOME=/opt/java/openjdk
+ARG GRADLE_HOME=/opt/gradle
+FROM gradle:jdk11 AS build
+
+FROM amazonlinux:2.0.20190508
+
+# Install Gradle and Java
+ARG JAVA_HOME
+ARG GRADLE_HOME
+
+ENV JAVA_HOME $JAVA_HOME
+ENV GRADLE_HOME $GRADLE_HOME
+
+COPY --from=build $JAVA_HOME $JAVA_HOME
```

Comment:
Could do with a comment to say this is being taken from the `FROM gradle:jdk11 AS build`

Reply:
It says `--from=build` and `AS build` at the top of the file - this is standard Dockerfile syntax. Do we really need a comment for that? I think it's clear from the context